fix: href-clickable

mfd2007 opened the pull request:
PR migrates changes in the template process from Secvisogram (see https://github.com/secvisogram/secvisogram/pull/511)

This PR isolates the changes to the mustache templates from #197 from other changes and replaces #197

Commit:
fix: href-clickable

rainer-exxcellent reviewed (commented):
At the moment, the URL is repeated in round brackets. Is this intentional?

<img width="729" height="213" alt="Image" src="https://github.com/user-attachments/assets/389da381-1ef9-4b47-9d74-8624fbc3b9df" />

mfd2007 committed:
Update Script.mjs

Copy line from https://github.com/secvisogram/secvisogram/blob/main/app/lib/app/SecvisogramPage/View/shared/HTMLTemplate.js

mfd2007 commented:
If there is still a difference to the preview in Secvisogram, please fix it.

rainer-exxcellent commented:
> If there is still a difference to the preview in Secvisogram, please fix it.

The round brackets are inserted via CSS selector:

https://github.com/secvisogram/csaf-cms-backend/blob/d6cc7654c30a95e4fbcb3888e8cbf8c9bd14ae01/src/main/resources/de/bsi/secvisogram/csaf_cms_backend/mustache/Template.html#L315

What was the initial intention of this Selector? It is still necessary?

rainer-exxcellent committed:
feat(mustache): #208 remove gutenberg css

rainer-exxcellent reviewed (approved):
LGTM
